platforms/web/lib/useWysiwyg.delete.test.tsx

```diff
@@ -77,8 +80,8 @@ describe('delete content', () => {
 
         // Then
         await waitFor(() => {
-            expect(textbox).toContainHTML('foar');
-            expect(textbox).toHaveAttribute('data-content', 'foar');
+            expect(textbox).toContainHTML('<p>foar</p>');
+            expect(textbox).toHaveAttribute('data-content', '<p>foar</p>');
         });
```

Comment:
No changes to these tests or behaviour, simply needed to update the fact they use `<br />` tags before and we've moved to paragraphs now